lib/bundler/templates/newgem/bin/setup.tt

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
+cd -
```

Comment:
Checked it at my Ubuntu and found out that cd - is Mac-only shotcut.

Updated it with much more portable and accurate version, borrowed from here:

http://stackoverflow.com/questions/3349105/how-to-set-current-working-directory-to-the-directory-of-the-script